.eslintrc.js

```diff
@@ -10,10 +10,7 @@ module.exports = {
             "error",
             2
         ],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
+        "linebreak-style": 0,
```

Comment:
This applies to SME's source code, which I'd prefer to keep standardized on unix line endings.

Reply:
What do you mean 'SME'? How about change it to
        "linebreak-style": [
            "error",
            process.platform === "win32" ? "windows" : "unix"
        ]?

Reply:
When I checked the code from 'source-map', I found it also use the 'unix' option. Maybe we can move the eslint check from 'test', and add a lint task. Just like
"scripts": {
    "test": "mocha",
    "lint": "eslint *.js"
 }.
So the test will not always fail under windows.